hoot-core-test/src/test/cpp/hoot/core/algorithms/changeset/ChangesetDeriverTest.cpp

```diff
@@ -27,9 +27,9 @@
 
```

Comment:
Moved changeset related code from the io namespace to algorithms/changeset, as I think its more appropriate.